app/controllers/api/v1/portfolios_controller.rb

```diff
@@ -2,18 +2,6 @@ module Api
   module V1
     class PortfoliosController < ApplicationController
       include Api::V1::Mixins::IndexMixin
```

Comment:
It may be good to also pull the filtering from `include Api::V1::Mixins::IndexMixin` into a Pundit scope as discussed ( in a follow up )